docs/excel/performance.md

```diff
@@ -71,11 +71,15 @@ _Where:_
 
 Please be aware that some of the “properties” under an object may have the same name as another object. For example, `format` is a property under range object, but `format` itself is an object as well. So, if you make a call such as `range.load("format")`, this is equivalent to `range.format.load()`, which is an empty load() call that can cause performance problems as outlined previously. To avoid this, your code should only load the “leaf nodes” in an object tree. 
 
-## Suspend calculation temporarily
+## Suspend Excel processes temporarily
 
-If you are trying to perform an operation on a large number of cells (for example, setting the value of a huge range object) and you don't mind suspending the calculation in Excel temporarily while your operation finishes, we recommend that you suspend calculation until the next ```context.sync()``` is called.
+When your add-in edits an Excel workbook through the JavaScript API, the experience is similar to when a user edits it through the UI. This can be slower than desired when dealing with large data sets, due to the side processes Excel runs. Some of these can be controlled to yield a performance benefit.
```

Comment:
I'm not convinced that the "experience is similar". Even if it is, do we need to say this?